hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/metric/SampleQuantilesTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.hadoop.ozone.metric;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.hadoop.metrics2.util.Quantile;
+import org.apache.hadoop.ozone.metrics.OzoneSampleQuantiles;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests for SampleQuantiles.
+ */
+class SampleQuantilesTest {
```

Comment:
Tests named `*Test` are not executed:

https://github.com/apache/ozone/blob/6f82e7b77c6463d3f9d2c9c967dc988b6e208066/pom.xml#L1844

Reply:
Thanks. Fixed.